content/en/docs/Tasks/update-and-ship-operator.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Update and ship an Operator"
+date: 2021-03-10
+weight: 2
+description: > 
+    In depth explanation of operator update and shipment process. 
+--- 
+
+## Introduction
+
+In the Operator Lifecycle Manager (OLM) ecosystem, the following resources are used to resolve Operator installations and upgrades:
+
+1. `ClusterServiceVersion (CSV)` - A YAML manifest created from Operator metadata that assists the Operator Lifecycle Manager (OLM) in running the Operator in a cluster.
+
+    A CSV is the metadata that accompanies an Operator container image, used to populate user interfaces with information like its logo, description, and version. It is also a source of technical information needed to run the Operator, like the RBAC rules it requires and which Custom Resources (CRs) it manages or depends on.
+
+    A CSV is composed of a Metadata, Install strategy, and CRDs.
+
+2. `CatalogSource` - Operator metadata, defined in CSVs, can be stored in a collection called a CatalogSource. CatalogSource contains metadata that OLM can query to discover and install operators with their dependencies.
+
+3. `Subscription` - A user indicates a particular package and channel in a particular CatalogSource in a Subscription. Subscription ensures OLM will manage and upgrades/installs the operator to ensure the latest version is always running in the cluster
+
+
+OLM uses CatalogSources, which use the Operator Registry API, to query for available Operators as well as upgrades for installed Operators.
+
+![CatalogSource Image](/docs/Tasks/images/catalogsource.png)
+
+`Figure 1. CatalogSource overview`
+
+In the above image, etcd is a package. Alpha and beta are the channels.
+
+Within a CatalogSource, Operators are organized into packages and streams of updates called channels, which should be a familiar update pattern from OpenShift Container Platform or other software on a continuous release cycle like web browsers.
+
+![Channels Image](/docs/Tasks/images/channels.png)
+
+`Figure 2. Packages and channels in a CatalogSource`
+
+A user indicates a particular package and channel in a particular CatalogSource in a Subscription.
```

Comment:
The channel is defined in the index image see: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/metadata/annotations.yaml#L7

Also, see here how to update the channels with sdk : https://master.sdk.operatorframework.io/docs/overview/cheat-sheet/#updating-bundle-channels

Reply:
Also, the channels might feet better in https://github.com/operator-framework/olm-docs/pull/136 and here we could only describe the options to configure the paths and the upgrade paths.